web-lib-funcs.pl

```diff
@@ -10676,6 +10676,16 @@ sub close_tempfile
 	# Closing a file
 	my $noerror = $main::open_tempfiles_noerror{$_[0]};
 	&webmin_debug_log("CLOSE", $_[0]) if ($gconfig{'debug_what_write'});
+	my $getfacl = &has_command("getfacl");
+	my $setfacl = &has_command("setfacl");
+	my $file_acls;
+	if ($getfacl && $setfacl) {
+		# Set original ACLs
+		$file_acls = &transname();
+		system("$getfacl --absolute-names ".
+			quotemeta($_[0]).">$file_acls");
```

Comment:
Can we do this by reading the current ACLs into a variable rather than a temp file?

Reply:
> Can we do this by reading the current ACLs into a variable rather than a temp file?

Yes, why not. Check this https://github.com/webmin/webmin/pull/2211/commits/82f5284ffca200e341111db21c552fd3480de60b patch out.